src/Microsoft.Graph.Core/Requests/BaseRequest.cs

```diff
@@ -138,6 +138,31 @@ public class BaseRequest : IBaseRequest
         }
 
         /// <summary>
+        /// Sends the multipart request.
```

Comment:
Yes, but only the checks for IsNullOrEmpty and whether the auth delegate was set. I left them as is and will refactor if we ever add another *Send method. I couldn't reuse the SendAsync and SendRequestAsync for a few reasons.